### ISSUE TITLE
[Bug]: Connect uses the default host directly instead of the passed alias's host

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#17858]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):  pymilvus-2.1.0.dev90
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Connect with alias get exception `pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530. Timeout)>`
```
my_host = "10.98.0.8"
connections.add_connection(a={"host": str(my_host), "port": "19530"})
connections.connect(alias='a')
```
The problem is that when the user calls `connect` and only passes the `alias` parameter, the `DefaultConfig.DEFAULT_HOST` is used directly in `connection.py`
![image](https://user-images.githubusercontent.com/27288593/176101548-57ec9f3d-5b2a-403a-aad3-b0645fb86ad4.png)

### Expected Behavior

Use the host of the alias to connect milvus server

**Comment by binbinlv** [comment]
And the same issue,

if connect with non-existed alias, it will connect with "localhost:19530" too.

**Comment by XuanYang-cn** [comment]
/unassign
/assign @ThreadDao @binbinlv 

Fixed at pymilvus==2.1.0.dev92, please help to verify

**Comment by binbinlv** [comment]
Verified and fixed in "2.1.0.dev92", close it.
